tools/test-case-generators/generate-test-cases

```diff
@@ -81,25 +100,104 @@ def main(distro, arch, image_types, keep_image_info, store, output):
+                            "name": "core"
+                        }
+                    ],
+                    "customizations": {
+                        "hosname": "my-host",
+                        "kernel": {
+                            "append": "debug"
+                        },
+                        "sshkey": [
+                            {
+                                "user": "user1",
+                                "key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC61wMCjOSHwbVb4VfVyl5sn497qW4PsdQ7Ty7aD6wDNZ/QjjULkDV/yW5WjDlDQ7UqFH0Sr7vywjqDizUAqK7zM5FsUKsUXWHWwg/ehKg8j9xKcMv11AkFoUoujtfAujnKODkk58XSA9whPr7qcw3vPrmog680pnMSzf9LC7J6kXfs6lkoKfBh9VnlxusCrw2yg0qI1fHAZBLPx7mW6+me71QZsS6sVz8v8KXyrXsKTdnF50FjzHcK9HXDBtSJS5wA3fkcRYymJe0o6WMWNdgSRVpoSiWaHHmFgdMUJaYoCfhXzyl7LtNb3Q+Sveg+tJK7JaRXBLMUllOlJ6ll5Hod root@localhost"
+                            }
+                        ],
+                        "user": [
+                            {
+                                "name": "user2",
+                                "description": "description 2",
+                                "password": "$6$BhyxFBgrEFh0VrPJ$MllG8auiU26x2pmzL4.1maHzPHrA.4gTdCvlATFp8HJU9UPee4zCS9BVl2HOzKaUYD/zEm8r/OF05F2icWB0K/",
+                                "key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC61wMCjOSHwbVb4VfVyl5sn497qW4PsdQ7Ty7aD6wDNZ/QjjULkDV/yW5WjDlDQ7UqFH0Sr7vywjqDizUAqK7zM5FsUKsUXWHWwg/ehKg8j9xKcMv11AkFoUoujtfAujnKODkk58XSA9whPr7qcw3vPrmog680pnMSzf9LC7J6kXfs6lkoKfBh9VnlxusCrw2yg0qI1fHAZBLPx7mW6+me71QZsS6sVz8v8KXyrXsKTdnF50FjzHcK9HXDBtSJS5wA3fkcRYymJe0o6WMWNdgSRVpoSiWaHHmFgdMUJaYoCfhXzyl7LtNb3Q+Sveg+tJK7JaRXBLMUllOlJ6ll5Hod root@localhost",
+                                "home": "/home/home2",
+                                "shell": "/bin/sh",
+                                "groups": [
+                                    "group1"
+                                ],
+                                "uid": 1020,
+                                "gid": 1050,
+                            }
+                        ],
+                        "group": [
+                            {
+                                "name": "group1",
+                                "gid": 1030
+                            },
+                            {
+                                "name": "group2",
+                                "gid": 1050
+                            }
+                        ],
+                        "timezone": {
+                            "timezone": "Europe/London",
+                            "ntpservers": [
+                                "time.example.com"
+                            ]
+                        },
+                        "locale": {
+                            "languages": [
+                                "en_US"
+                            ],
+                            "keyboard": "dvorak"
+                        },
+#                        "firewall": {
```

Comment:
why is this commented out?

Reply:
The firewall stage requires firewalld to be installed in the image (sad, but also fine). Sadly, we cannot currently install the `firewalld` packake on all our image types: https://github.com/osbuild/osbuild-composer/issues/806

Reply:
Ah, interesting, let's add this later then.